free-programming-interactive-tutorials-en.md

```diff
@@ -165,6 +166,21 @@
 * [Learn AngularJS with free interactive lessons](http://www.learn-angular.org)
 
 
+#### Bootstrap
+
+* [Front End Libraries: Bootstrap](https://www.freecodecamp.org/learn/front-end-libraries/bootstrap)
+
+
+#### jQuery
+
+* [Front End Libraries: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/jquery)
+
+
+#### React.js
```

Comment:
This is more commonly known as React, and is categorised this way in all other documents. Could we use React instead of React.js?

```suggestion
#### React
```